public/data/jainpatel.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Jain Patel",
+    "type": "personal",
+    "bio": "learning code. I am open to learn new skills",
+    "avatar": "",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/jainpatel25",
+        "icon": "github"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "focus on web dev and opensource",
+        "date": "October 2022",
+        "icon": "",
```

Comment:
Use a valid icon here from the list of icons available in our docs [icon.md](https://github.com/EddieHubCommunity/LinkFree/blob/main/icons.md)